compiler/rustc_session/messages.ftl

```diff
@@ -133,7 +133,8 @@ session_unstable_virtual_function_elimination = `-Zvirtual-function-elimination`
 session_unsupported_crate_type_for_target =
     dropping unsupported crate type `{$crate_type}` for target `{$target_triple}`
 
-session_unsupported_dwarf_version = requested DWARF version {$dwarf_version} is greater than 5
+session_unsupported_dwarf_version = requested DWARF version {$dwarf_version} is not supported
+session_unsupported_dwarf_version_help = valid DWARF versions are 2 - 5
```

Comment:
I don't want to nitpick but we don't seems to be using `valid` that much in error messages, instead we use `supported` which I think is also a better description since DWARF1 does exist and is valid (in a sense).

We should also maybe avoid ranges, as it may be clear whenever it's an included or excluded range, better IMO to list the 4 options.

```suggestion
session_unsupported_dwarf_version_help = supported DWARF versions are: 2, 3, 4 and 5
```

Reply:
Good observation, thanks for the feedback!